src/multihash.rs

```diff
@@ -173,6 +173,45 @@ impl<const S: usize> Multihash<S> {
             .expect("writing to a vec should never fail");
         bytes
     }
+
+    /// Truncates the multihash to the given size. It's up to the caller to ensure that the new size
+    /// is safe to use.
+    ///
+    /// If the new size is larger than the current size, this method does nothing.
+    ///
+    /// ```
+    /// use multihash::{Code, MultihashDigest};
+    ///
+    /// let hash = Code::Sha3_256.digest(b"Hello world!").truncate(20);
+    /// ```
+    pub fn truncate(&self, size: u8) -> Self {
+        let mut mh = *self;
+        mh.size = mh.size.min(size);
+        mh
+    }
+
+    /// Resizes the backing multihash buffer. If the multihash doesn't fit, this function fails and
```

Comment:
I didn't get what "doesn't fit" means when I read the comment. It fails, when the multihash is larger then the new size. Perhaps it can be rephrased?

Reply:
LGTY?